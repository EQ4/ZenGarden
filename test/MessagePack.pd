#N canvas 369 154 450 300 10;
#X obj 156 41 bng 15 250 50 0 empty empty empty 17 7 0 10 -262144 -1
-1;
#X obj 202 41 loadbang;
#X obj 187 178 pack f f f;
#X obj 187 219 print;
#X obj 184 95 t b b b;
#X obj 263 134 f 3;
#X obj 223 135 f 2;
#X obj 190 136 f 1;
#X connect 0 0 4 0;
#X connect 1 0 4 0;
#X connect 2 0 3 0;
#X connect 4 0 7 0;
#X connect 4 1 6 0;
#X connect 4 2 5 0;
#X connect 5 0 2 2;
#X connect 6 0 2 1;
#X connect 7 0 2 0;
