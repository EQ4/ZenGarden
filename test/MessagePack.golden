[@ 0.000ms] print: 1 2 3